### ISSUE TITLE
Encoding error: NullPointerException: Cannot invoke "java.lang.Long.longValue()" because the return value of "java.util.LinkedHashMap.get(Object)" is null

### ISSUE THREAD

**Issue by springmeyer** [maplibre/maplibre-tile-spec#75]
> java -jar ./build/libs/mvt2mlt-1.0-SNAPSHOT.jar -mvt ../test/fixtures/amazon/11_1037_704.pbf -decode

Gives:

```
java.lang.NullPointerException: Cannot invoke "java.lang.Long.longValue()" because the return value of "java.util.LinkedHashMap.get(Object)" is null
	at com.mlt.converter.mvt.MvtUtils.lambda$getIds$2(MvtUtils.java:109)
	at java.base/java.util.stream.ReferencePipeline$5$1.accept(ReferencePipeline.java:231)
	at java.base/java.util.ArrayList$ArrayListSpliterator.forEachRemaining(ArrayList.java:1625)
	at java.base/java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:509)
	at java.base/java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:499)
	at java.base/java.util.stream.ReduceOps$ReduceOp.evaluateSequential(ReduceOps.java:921)
	at java.base/java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234)
	at java.base/java.util.stream.ReferencePipeline.collect(ReferencePipeline.java:682)
	at com.mlt.converter.mvt.MvtUtils.getIds(MvtUtils.java:110)
	at com.mlt.converter.mvt.MvtUtils.decodeMvt(MvtUtils.java:48)
	at com.mlt.converter.mvt.MvtUtils.decodeMvt(MvtUtils.java:24)
	at com.mlt.converter.MltCliAdapter.main(MltCliAdapter.java:189)
```

**Comment by springmeyer** [comment]
This error appears to happen on all the amazon tiles

**Comment by springmeyer** [comment]
This specific exception is fixed with https://github.com/maplibre/maplibre-tile-spec/pull/111.  Now other errors are revealed that are in a separate code path:

```
java -jar ./build/libs/mvt2mlt-1.0-SNAPSHOT.jar -mvt ../test/fixtures/amazon/11_1037_704.pbf -decode -advanced
Time elapsed for encoding: 1026 milliseconds
java.lang.ArrayIndexOutOfBoundsException: Index 794 out of bounds for length 794
	at com.mlt.decoder.GeometryDecoder.getLineString(GeometryDecoder.java:478)
	at com.mlt.decoder.GeometryDecoder.getLinearRing(GeometryDecoder.java:455)
	at com.mlt.decoder.GeometryDecoder.decodeGeometry(GeometryDecoder.java:128)
	at com.mlt.decoder.MltDecoder.decodeMlTile(MltDecoder.java:68)
	at com.mlt.converter.MltCliAdapter.main(MltCliAdapter.java:239)
java.lang.ArrayIndexOutOfBoundsException: Index 800 out of bounds for length 794
	at com.mlt.decoder.GeometryDecoder.getLineString(GeometryDecoder.java:478)
	at com.mlt.decoder.GeometryDecoder.getLinearRing(GeometryDecoder.java:455)
	at com.mlt.decoder.GeometryDecoder.decodeGeometry(GeometryDecoder.java:128)
	at com.mlt.decoder.MltDecoder.decodeMlTile(MltDecoder.java:68)
	at com.mlt.converter.MltCliAdapter.main(MltCliAdapter.java:239)
java.lang.ArrayIndexOutOfBoundsException: Index 166 out of bounds for length 166
	at com.mlt.decoder.GeometryDecoder.getLineString(GeometryDecoder.java:478)
	at com.mlt.decoder.GeometryDecoder.getLinearRing(GeometryDecoder.java:455)
	at com.mlt.decoder.GeometryDecoder.decodeGeometry(GeometryDecoder.java:179)
	at com.mlt.decoder.MltDecoder.decodeMlTile(MltDecoder.java:68)
	at com.mlt.converter.MltCliAdapter.main(MltCliAdapter.java:239)
Time elapsed for decoding: 38 milliseconds
```

**Comment by springmeyer** [comment]
This is fixed by #111
